Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidationRepository.kt

```diff
@@ -107,11 +119,34 @@ class DccValidationRepository @Inject constructor(
             DccValidationData(
                 countries = newCountryData,
                 acceptanceRules = newAcceptanceData,
-                invalidationRules = newInvalidationData
+                invalidationRules = newInvalidationData,
+                boosterNotificationRules = this.boosterNotificationRules
```

Comment:
Can we move it into it own repository? `Booster rules` and `Validation rules` are used in two different use cases, I would keep them independently to avoid any delay to load all kind of rules using the same repository. what do you think?

Reply:
yes we can! make sense